docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,791 @@ thor:
+
+          <para>You should see a result similar to this, possibly with some additional headers:</para>
+
+          ```xml
+          <Result>
+            <Dataset name='localHttpEchoResult'>
+              <Row>
+                <Method>GET</Method>
+                <UrlPath>/WsSmc/HttpEcho</UrlPath>
+                <UrlParameters>name=doe,joe&amp;number=1</UrlParameters>
+                <Headers>
+                  <Header>Accept-Encoding: gzip, deflate</Header>
+                  <Header>Accept: text/xml</Header>
+                </Headers>
+              </Row>
+            </Dataset>
+          </Result>
+          ```
+        </sect4>
 
-    <sect2 id="HPCCContainer_Certificates" role="brk">
-      <title>Certificates</title>
+        <sect4 id="RemoteMutualTLS">
+          <title>Remote Mutual TLS</title>
 
-      <para>HPCC Systems containerized deployments support comprehensive
-      certificate management through Helm chart configuration. This includes
-      support for multiple certificate issuers, domain management, and
-      advanced certificate features for secure communication between
-      components.</para>
+          <para>This walkthrough demonstrates using a single Hashicorp Vault
+          PKI Certificate authority to establish a zone of trust between two separate
+          HPCC environments. cert-manager continues to handle the backing Kubernetes
+          resources. For this example each HPCC environment is in a separate kubernetes
+          namespace: hpcc1 and hpcc2, but in production they could be in different
+          Kubernetes clusters, or even in different cloud subscriptions.</para>
 
-      <para>The certificate system integrates with cert-manager to provide
-      automated certificate generation and renewal for both internal (local,
-      i.e., communication between HPCC components within the same cluster or
-      namespace) and external (public/remote) communications.</para>
+          <para>
+          Here we have hpcc2.roxie making a call to hpcc1.roxie in another 
+          environment, and we want them to trust each other. There are two main configuration
+          customizations needed to set up this zone
+          of trust. Each of the two HPCC environments have these changes.</para> 
 
-      <sect3 id="Container_Certificates">
-        <title>Enable Certificates</title>
+          <para>First, the remote issuer is enabled and its spec set to use the Vault.
+          Both HPCC environments share the same Vault server, which allows them to use
+          the same underlying certificates to establish trush. Note alseo that the
+          <emphasis>ca</emphasis> property must be set to null to override the default.</para>
 
-        <para>Use the certificates section to enable cert-manager to generate
-        TLS certificates for each component in the HPCC Systems
-        deployment.</para>
+          <programlisting>
```

Comment:
comment : notice this section properly using programlisting element to denote the code example